src/assets/i18n/en.json

```diff
@@ -22,6 +22,7 @@
   "assignment.status.reply": "Provide answer",
   "assignment.status.user_company_hire": "The company wants to hire you",
   "assignment.status.you_hired": "You've been hired",
+  "assignment.comments.title": "Comment on the job",
```

Comment:
Good question! I was actually planning for us to sit down and walk through it! I think the entire dev-team should now how we handle it (roughly) while you and I should be pros 😉 Lets look at it real quick today!